File: android/app/src/main/java/com/phicdy/totoanticipation/view/GameListView.java
package com.phicdy.totoanticipation.view;

import android.support.annotation.NonNull;

public interface GameListView {
    void initList();
    void setTitleFrom(@NonNull String xxTh, @NonNull String deadline);
    void startProgress();
    void stopProgress();
    void startTotoAnticipationActivity(@NonNull String totoNum);
    void goToSetting();
}

File: android/app/src/main/java/com/phicdy/totoanticipation/view/GameListView.kt
package com.phicdy.totoanticipation.view

interface GameListView {
    fun initList()
    fun setTitleFrom(xxTh: String, deadline: String)
    fun startProgress()
    fun stopProgress()
    fun startTotoAnticipationActivity(totoNum: String)
    fun goToSetting()
    fun showAnticipationStart()
    fun showAnticipationFinish()
    fun notifyDataSetChanged()
    fun showAnticipationNotSupport()
    fun showPrivacyPolicyDialog()
}
